workflow-templates/im-deploy-nuget-package.yml

```diff
@@ -73,7 +65,7 @@ jobs:
       #   run: dotnet pack --configuration Release
 
       # - name: Pack with version from env.Tag
-      #   run: dotnet pack --configuration Release -p:PackageVersion=${{ env.TAG }}
+      #   run: dotnet pack --configuration Release -p:PackageVersion=${{ steps.increment.outputs.VERSION }}
```

Comment:
Should the name be updated to not reference env.Tag?

Reply:
yeah I'll do that.  I think I need to do some re-work here.  The nugets/npms were done before the deployment re-work so I think this could benefit from that.  It's still different though so I'm not sure how I want to handle it yet.  This one was very alternative heavy so there were lots of things to change.

Reply:
I guess I misunderstood the question many times...you mean like 67?

Reply:
Sorry that my comment was confusing! But yes, I figured line 67 would be confusing since it _looks_ like a different value is being used on 68 (even though they're really the same thing).